pkg/apis/admission/types.go

```diff
@@ -91,7 +91,7 @@ type AdmissionResponse struct {
 	// This field IS NOT consulted in any way if "Allowed" is "true".
 	// +optional
 	Result *metav1.Status
-	// Patch contains the actual patch. Currently we only support a response in the form of JSONPatch, RFC 6902.
+	// Patch contains the actual patch, base-64 encoded. Currently we only support a response in the form of JSONPatch, RFC 6902.
```

Comment:
documentation is surfaced for many different contexts - protobuf, go types, and API documentation. the base64 encoding only applies to json/yaml encodings, so should note that. (comment applies to all instances)